modules/govuk/templates/asset_manager_extra_nginx_config.conf.erb

```diff
@@ -71,6 +70,10 @@ location ~ /cloud-storage-proxy/(.*) {
   proxy_hide_header x-amz-replication-status;
   proxy_hide_header x-amz-meta-md5-hexdigest;
 
+  # Strip any Authorization header from the client, as this will cause
+  # S3 to return an error.
+  proxy_set_heasder Authorization "";
```

Comment:
I'm guessing this should actually be `proxy_set_header` :)

Reply:
Oh man